docs/sources/writing-guide/front-matter/index.md

```diff
@@ -70,6 +73,44 @@ They do not appear in the resulting HTML source for the page and do not affect S
 
 Ideally, use single terms as opposed to phrases.
 
+### labels
+
+Used to add labels that appear before the topic title.
+Only certain labels are supported.
+
+For `labels.products`, the supported values and the resulting website labels are:
+
+- `cloud`: "Grafana Cloud"
+- `enterprise`: "Enterprise"
+- `oss`: "Open source"
+
+Labels can be inherited through cascading front matter.
+Each project has a set of default labels defined in the root `_index.md` file for the project.
+
+For versioned projects, the `_index.md` file resides in the website repository.
+For unversioned projects, the `_index.md` file resides in the project repository.
+
+If the default labels are incorrect for a page, or directory of pages, you should update the labels.
```

Comment:
I feel like it would be good to say that basically, you mostly don't have to worry about labels unless you think the existing page labels are incorrect/insufficient or if you're adding a new directory.

Reply:
Good idea, I'll add that in.